README.md

```diff
@@ -1,2 +1,13 @@
 # httpd
 Hypertext Transfer Protocol Daemon
```

Comment:
Missing newline between sections if sections are to be separated by empty newlines.